metpy/io/upperair.py

```diff
@@ -155,6 +163,16 @@ def get_data(time, site_id, region='naconf'):
                              'for station {stid}.'.format(time=time, region=region,
                                                           stid=site_id))
 
+        # See if the return is valid, but has no data
+        if data.find(b'too busy') != -1:
+            if retries > 0:
+                sleep(retry_delay / 1000.)
+                WyomingUpperAir.get_data(time, site_id, region, retries - 1)
```

Comment:
1. Need to pass `retry_delay` to the recursive call.
2. I'm pretty sure that should be: `return WyomingUpperAir.get_data(...)`, otherwise no data

The latter makes me nervous about a lack of tests....but I'm not clear how best to add one.